packages/big-design/src/components/List/Item/styled.tsx

```diff
@@ -13,6 +13,8 @@ export const StyledListItem = styled.li<ListItemProps>`
   min-width: ${({ theme }) => theme.helpers.remCalc(256)};
   outline: none;
   padding: 0 ${({ theme }) => theme.spacing.medium};
+  transition: all 150ms ease-out;
```

Comment:
Shouldn't need all if using `transition-property`
```suggestion
  transition: 150ms ease-out;
```